tools/test.py

```diff
@@ -188,6 +170,49 @@ def main():
         distributed = True
         init_dist(args.launcher, **cfg.dist_params)
 
+    test_dataloader_default_args = dict(
```

Comment:
can we extract a function so that the function can be reused in train and test?

Reply:
The default keys for the different loader is different